src/questions/aave-other-features.yaml

```diff
@@ -285,11 +285,11 @@
     - aave-gho
   difficultyLevel: "Medium"
   choices:
-    - content: It will be set by the AaveDAO initially
+    - content: It will be set by the Aave DAO initially
       key: A
-    - content: The rates determined by AaveDAO will be final and cannot be changed
+    - content: The rates determined by Aave DAO will be final and cannot be changed
       key: B
-    - content: It will be stable rate as opposed to variable rate, can be adapted to market conditions
+    - content: It will be stable rate, it can be changed 
```

Comment:
```suggestion
    - content: It will be stable rate but can be changed to variable rate in future.
```

Reply:
Robin as of my understanding the stable rate could be changed from x% to y%, not from stable to variable